packages/commonwealth/client/scripts/views/components/NewThreadForm/NewThreadForm.tsx

```diff
@@ -48,19 +47,7 @@ export const NewThreadForm = () => {
 
   const topicsForSelector = topics?.reduce(
     (acc, t) => {
-      if (featureFlags.newGatingEnabled) {
-        acc?.enabledTopics?.push(t);
-      } else {
-        if (
-          isAdmin ||
-          t.tokenThreshold.isZero() ||
-          !app.chain.isGatedTopic(t.id)
-        ) {
-          acc?.enabledTopics?.push(t);
-        } else {
-          acc?.disabledTopics?.push(t);
-        }
-      }
+      acc?.enabledTopics?.push(t);
       return acc;
     },
     { enabledTopics: [], disabledTopics: [] },
```

Comment:
maybe we could do 
```
const topicsForSelector =  {
  enabledTopics: topics
}
```
or just `topicsForSelector = topics` and update references.